manifest.json

```
@@ -10151,4 +10205,4 @@
       ]
     }
   ]
-}
+}
```

nit: add back line ending
```suggestion
}

```

---

OK, but for the record I *did* just "apply file formatting in VS Code" :)